run.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/bash
 set -e
+
+# wait for at most 30s for the db to be up
+/root/wait-for-it.sh ${POSTGRES_HOST}:${POSTGRES_PORT} -t 30
```

Comment:
:+1: for keeping the wait invocation close to the reason for waiting (that this script needs the DB to be up and running)